src/slack/events/message.ts

```diff
@@ -19,6 +19,17 @@ function register(bolt: App): void {
       logger.error('Something went wrong messaging the user...', err);
     }
   });
+
+  bolt.event('app_mention', ({ event, say }) => {
```

Comment:
Let's move this out into `appMention.ts`. Make sure to include that file within `events/index.ts` so it can register with the app.